var/spack/repos/builtin/packages/py-nibabel/package.py

```diff
@@ -13,8 +13,11 @@ class PyNibabel(PythonPackage):
     pypi = "nibabel/nibabel-3.2.1.tar.gz"
     git = "https://github.com/nipy/nibabel"
 
-    license("PDDL-1.0")
+    maintainers("ChristopherChristofi")
 
+    license("MIT")
```

Comment:
Thanks for adding yourself as maintainer! I've just one question regarding the license:

Besides MIT, it adds a few other Licenses for 3rd party components that it says are included:
https://github.com/nipy/nibabel/blob/master/COPYING

Could you clarify that? Is this package only installing the MIT based code and none of the 3rd party components?

Reply:
Hello @bernhardkaindl, thank you for the review, yes the installation does include the 3rd party components, I have updated the license directive to reflect their respective licensing. I hope this helps